docs/variants/asus_kgpe_d16/setup.md

```diff
@@ -1,7 +1,5 @@
 # RTE v.0.5.3 with Asus KGPE-D16 connection
 
-=========================================
```

Comment:
@miczyg1 I have no idea why it was added

Reply:
Leftover from gitlab 3mdeb/rte/docs, can be removed.

Reply:
@miczyg1 I removed it, do you approve my changes?

Reply:
This is obvious, so I'm resolcing it